canvas_modules/harness/test_resources/parameterDefs/toggle_paramDef.json

```diff
@@ -0,0 +1,507 @@
+{
+  "titleDefinition": {
+    "title": "toggle Fields"
+  },
+  "current_parameters": {
+    "string": "toggle",
```

Comment:
You're passing this value in paramDef. But I don't see this anywhere on UI. 
Since toggle is a boolean control, what does it have string value?

![Screen Shot 2022-07-28 at 1 46 57 PM](https://user-images.githubusercontent.com/25124000/181634531-2343640d-f61b-4169-bb36-252ad60f4da6.png)